node/caching-compiler.js

```diff
@@ -246,10 +250,7 @@ function readPkgInfo(dir) {
           for (let i = 0; i < filesCount; ++i) {
             // Later we'll change the value to the actual contents of the
             // file, but for now we merely register that it exists.
-            const file = cacheFiles[i];
-            if (/\.js$/.test(file)) {
-              pkgInfo.cache[file] = true;
-            }
```

Comment:
I think this is okay because we write `.js` cache files even if the original file was `.mjs`.